manifests/augeas.pp

```diff
@@ -1,40 +1,43 @@
-# == Definition jenkins::augeas
+# @summary Change config files using augeas
 #
-# === Parameters:
+# @param config_filename
+#   Filename of the configuration file to work on relative to the jenkins
+#   localstatedir.
```

Comment:
I prefer it without a linebreak within the description, but as long as puppet-strings renders it properly I'm happy.

Reply:
I always wrap my parameter documentation to a reasonable line length. In this case it may be more readable if I don't, but if there are more words ...